CHANGELOG.md

```diff
@@ -2,6 +2,32 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [2.51.0](https://github.com/serverless/serverless/compare/v2.50.0...v2.51.0) (2021-07-06)
+
+### Features
+
+- **AWS Lambda:** Add support for `SASL/PLAIN` auth to `kafka` event ([#9666](https://github.com/serverless/serverless/pull/9666)) ([3e14f06](https://github.com/serverless/serverless/commit/3e14f063052385026425021379bfc883dac5ff74)) ([Daniele Iasella](https://github.com/overbit))
```

Comment:
I think it'll be good to put it as `AWS Kafka`, as normally we create a commit scope for each event (For `kafka` we currently miss in https://github.com/serverless/serverless/blob/0c5b8dd831bcde80628c2ac548172ffaa9ce9ca6/commitlint.config.js but it's worth to add)